src/main/java/com/treasuredata/client/model/TDSavedQueryStartRequest.java

```diff
@@ -0,0 +1,36 @@
+package com.treasuredata.client.model;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.google.common.base.Optional;
+import org.immutables.value.Value;
+
+import java.util.Date;
+
+@Value.Immutable
+@Value.Style(visibility = Value.Style.ImplementationVisibility.PACKAGE)
+@JsonSerialize(as = ImmutableTDSavedQueryStartRequest.class)
+@JsonDeserialize(as = ImmutableTDSavedQueryStartRequest.class)
+public abstract class TDSavedQueryStartRequest
+{
+    public abstract String name();
+    public abstract Date scheduledTime();
+    public abstract Optional<Integer> num();
+    public abstract Optional<String> domainKey();
+
+    public interface Builder
+    {
+        Builder name(String name);
+        Builder scheduledTime(Date scheduledTime);
+        Builder num(Optional<Integer> num);
+        Builder num(int num);
```

Comment:
Yeah, the TD REST api documentation is a bit lacking. It doesn't even describe the  `/v3/schedule/run/` endpoint.